nautobot/docs/development/getting-started.md

```diff
@@ -120,10 +120,20 @@ git pull upstream develop
 !!! warning
     If you do not do this, you run the risk of having merge conflicts in your branch, and that's never fun to deal with. Trust us on this one.
 
-Now that you've got the latest upstream changes, create your branch. It's convention to always prefix your branch name with your GitHub username or your initials, and suffix it with the issue number if appropriate, separated by hyphens. For example:
+Now that you've got the latest upstream changes, create your branch. Whether you're creating a branch off a fork or working against the Nautobot origin repo, you should follow this convention for naming your branch: `u/yourusername-0000-branch-summary`, where `0000` is the related GitHub issue number. For example:
```

Comment:
To be clear, I presume `yourusername` should be your GitHub user account name?

Reply:
We use `yourusername` elsewhere in this doc with the assumed idea that that's your Github username but I will clarify.